components/raftstore/src/store/worker/compact.rs

```diff
@@ -169,6 +169,7 @@ impl Runnable<Task> for Runner {
                                 );
                             }
                         }
+                        fail_point!("raftstore::compact:CheckAndCompact");
```

Comment:
Maybe `raftstore::compact::CheckAndCompact::AfterCompact` is better?

Reply:
Ok. I will fix it. And I think it is better that other fail-point also be named as this. `{{module}}::{{file}}::{{method}}::{{point}}`.